include/kernel/thread.h

```diff
@@ -307,5 +307,20 @@ void awaken(void);
 /* Place the thread bound to this scheduling context in the release queue
  * of periodic threads waiting for budget recharge */
 void postpone(sched_context_t *sc);
+
+static inline ticks_t getNextInterrupt(void) {
```

Comment:
Where does this come from again?

Reply:
Similar to setNextInterrupt but omits setDeadline unless it must be done. The same as Anna's fastpath:
https://github.com/pingerino/seL4/blob/6ec7cc15a92b7c6aa93504104321f9cd56dfe7c8/include/kernel/thread.h#L316